scripts/localnet.sh

```diff
@@ -25,7 +21,16 @@ echo_success () {
 }
 
 # Set localnet settings
-BINARY=./build/archwayd
+if [[ -f "build/archwayd" ]] ;then
+  BINARY=build/archwayd
+  # Console log text colour
+  red=`tput setaf 9`
+  green=`tput setaf 10`
+  blue=`tput setaf 12`
+  reset=`tput sgr0`
+else
+  BINARY=archwayd
```

Comment:
We cannot rely on that since the containers don't have  it in `build/archwayd` 

You cannot run it with `latest` tag since we don't have that. Please use it as is.

Reply:
Sure it works for docker but for `make run` target it might inconsisten behaviour if the `build/archwayd` does not exist and the installed version is different.

At the very least we should add `make build` as a dependency for `make run` [here](https://github.com/archway-network/archway/pull/439/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R271) to make sure `build/archwayd ` exist when running `make run` or `make run-continue` targets

Reply:
I think that is outside the scope of this PR.
Makes sense to have it but I think in a different PR.